lib/models/instance.js

```diff
@@ -349,6 +343,33 @@ Instance.prototype.getElasticHostname = function () {
   });
 };
 
+/**
+ * Get name for current branch
+ * @return {String}
+ */
+Instance.prototype.getBranchName = function () {
+  return keypather.get(this.contextVersion, 'getMainAppCodeVersion().attrs.branch');
+};
+
+/**
+ * Get name for current repository
+ * @return {String}
+ */
+Instance.prototype.getRepoName = function () {
+  var repo = keypather.get(this.contextVersion, 'getMainAppCodeVersion().attrs.repo');
+  if (repo) {
+    return repo.split('/')[1];
+  }
+};
+
+/**
+ * Get `repo/branch` style string for instance
+ *
+ * Function here for backwards compatibility. Use `getNameAndBranchName` instead
+ *
+ * @deprecated
```

Comment:
> `@deprecated`

That's a lousy way to deprecate something. Why do you want to get rid of this function (`getRepoAndBranchName`) anyway? This is more correct than the one you created below, `getInstanceAndBranchName`, if you're trying to get the `repository/branch` for which this was created